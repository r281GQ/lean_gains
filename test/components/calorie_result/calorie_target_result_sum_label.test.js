import React from 'react';
import { shallow, render, mount } from 'enzyme';
import CalorieTargetResultSumLabel from './../../../src/components/calorie_result/calorie_target_result_sum_label';

describe('calorie_target_result_sum_label', () => {
  it('should render label with given properties', () => {
    const component = shallow(
      <CalorieTargetResultSumLabel
        label="rest"
        calorieTarget={{ rest: { calorie: 1500 } }}
      />
    );
    expect(component).toMatchSnapshot();
  });
});
