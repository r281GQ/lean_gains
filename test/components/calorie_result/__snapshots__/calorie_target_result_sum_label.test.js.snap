// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`calorie_target_result_sum_label should render label with given properties 1`] = `
<div
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <FlatButton
    disabled={true}
    fullWidth={false}
    label="Calculated calories for rest day: 1500"
    labelPosition="after"
    labelStyle={Object {}}
    onKeyboardFocus={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onTouchStart={[Function]}
    primary={false}
    secondary={false}
  />
</div>
`;
